Sources/PCRoomDelegate.swift

```diff
@@ -10,17 +10,15 @@ public protocol PCRoomDelegate: NSObjectProtocol {
     func userJoined(user: PCUser)
     func userLeft(user: PCUser)
 
-    func userCameOnlineInRoom(user: PCUser)
-    func userWentOfflineInRoom(user: PCUser)
+    func userPresenceChanged(previous: PCPresenceState, current: PCPresenceState, user: PCUser)
 
     func newCursor(cursor: PCCursor)
 
     // TODO: This seems like it could instead be `userListUpdated`, or something similar?
     func usersUpdated()
 
     // TODO: Make all of this work, and probably duplicate in PCChatManagerDelegate
-
-    //    func subscriptionStateChanged(from: PPResumableSubscriptionState, to: PPResumableSubscriptionState)
+//    func subscriptionStateChanged(from: PPResumableSubscriptionState, to: PPResumableSubscriptionState)
```

Comment:
It can probably just be removed for now. Originally there was a `connectionState` property on `ChatManager` but it got cut a long time ago. I don't think there's a pressing need for it right now so I'll clean up (remove) the comments